samples/zigbee/ncp/README.rst

```diff
@@ -116,17 +116,34 @@ This can be configured by setting:
 * :kconfig:`CONFIG_ZBOSS_TRACE_USB_CDC_LOGGING` - to select nRF USB serial over UART serial.
 * :kconfig:`CONFIG_ZBOSS_TRACE_LOGGER_DEVICE_NAME` - to select the serial device for printing Zigbee stack logs, set to ``"CDC_ACM_1"``.
 
-To create two instances of USB CDC ACM for application, set the following:
+To create two instances of USB CDC ACM for application, complete the following steps:
```

Comment:
I'm being a bit picky here, but it's not clear to me why you need to create a second instance. The 117 line only suggests 1 of these 2 instances...

Reply:
One for NCP communication and one for ZBOO trace messages